nginx-proxy/nginx.conf

```diff
@@ -8,21 +8,24 @@ http {
     add_header 'X-wc-nginx-proxy' 'true';
 
     # articles
-    rewrite ^/\d+\/\d+\/\d+\/(.*)/$ https://next.wellcomecollection.org/articles/$1 redirect;
+    rewrite ^/\d+\/\d+\/\d+\/(.*)/?$ https://next.wellcomecollection.org/articles/$1 redirect;
 
     # categories
-    rewrite ^/category\/(.*)/$ https://next.wellcomecollection.org/articles?q=category:$1 redirect;
+    rewrite ^/category\/(.*)/?$ https://next.wellcomecollection.org/articles?q=categories:$1 redirect;
 
     # tags
-    rewrite ^/tag\/(.*)/$ https://next.wellcomecollection.org/articles?q=tag:$1 redirect;
+    rewrite ^/tag\/(.*)/?$ https://next.wellcomecollection.org/articles?q=tags:$1 redirect;
 
     # pages
-    rewrite ^/about/$ https://wellcomecollection.org/what-we-do/about-wellcome-collection redirect;
-    rewrite ^/about/terms-and-conditions/$ https://wellcome.ac.uk/about-us/terms-use redirect;
+    rewrite ^/about/?$ https://wellcomecollection.org/what-we-do/about-wellcome-collection redirect;
+    rewrite ^/about/terms-and-conditions/?$ https://wellcome.ac.uk/about-us/terms-use redirect;
```

Comment:
We have the URL:
`https://blog.wellcomecollection.org/about/terms-and-conditions/`

Which will redirect to:
`https://wellcome.ac.uk/about-us/terms-use redirect`